src/meson.build

```diff
@@ -72,17 +72,22 @@ libepoxy = library(
   link_args: common_ldflags,
 )
 
+epoxy_has_glx = build_glx ? '1' : '0'
+epoxy_has_egl = build_egl ? '1' : '0'
+epoxy_has_wgl = build_wgl ? '1' : '0'
+
 libepoxy_dep = declare_dependency(
   link_with: libepoxy,
   include_directories: libepoxy_inc,
   dependencies: epoxy_deps,
   sources: epoxy_headers,
+  variables: {
```

Comment:
The usage of a dictionary might require a bump in the required version of Meson. Right now it's at 0.48, but I'm amenable to changing it to the minimum version that supports this.

Reply:
Good point, it actually requires 0.54, updated the PR.

I'm actually disappointed that declare_dependency() takes variables as dict and pkg.generate() takes variables as list of "var=val", which means I had to duplicate that information in this PR. Ideally meson should accept both formats so we could have a common variable between boths.